apps/dashboard/test/system/files_test.rb

```diff
@@ -65,19 +65,16 @@ class FilesTest < ApplicationSystemTestCase
       assert_selector '#directory-contents tbody tr', count: 1
       find('#clipboard-copy-to-dir').click
 
-      # if this fails it is due to the directory table not reloading
-      # assert_selector '#directory-contents tbody tr', count: 3, wait: 10
-      assert_selector 'tbody a', exact_text: 'app', wait: MAX_WAIT
-      
-      # we need to wait due to the nature of how the "% copy files" displays
-      assert_selector 'span', text: 'copy files', count: 1, wait: MAX_WAIT 
-      
+      # files are copying but it takes a little while
+      find('tbody a', exact_text: 'app', wait: MAX_WAIT)
+      find('tbody a', exact_text: 'config', wait: MAX_WAIT)
+      find('tbody a', exact_text: 'manifest.yml', wait: MAX_WAIT)
+
+      # with copying done, let's assert on the UI and the file system
+      assert_selector 'span', text: '100% copy files', count: 1
       assert_equal "", `diff -rq #{File.join(dir, 'app')} #{Rails.root.join('app').to_s}`.strip, "failed to recursively copy app dir"
       assert_equal "", `diff -rq #{File.join(dir, 'config')} #{Rails.root.join('config').to_s}`.strip, "failed to recursively copy config dir"
       assert_equal "", `diff -q #{File.join(dir, 'manifest.yml')} #{Rails.root.join('manifest.yml').to_s}`.strip, "failed to copy manifest.yml"
-      
```

Comment:
@johrstrom .  Did you not want to check to ensure the "100% copy files" disappears?

Reply:
I can try, but had to add a `sleep` here. I guess it's a balance between reliability (the test is always deterministic) and coverage.

Reply:
You didn't have to add it back. I was just wondering why?

Reply:
Coverage is good. I'm OK with the sleep - until it starts breaking